### DISCUSSION TITLE
Offer a standard theme

### DISCUSSION THREAD

**Discussion by thgh** [payloadcms/payload#1280] [Feature Requests & Ideas]
Popular Wordpress [page](https://wpbakery.com/) [builder](https://elementor.com/) plugins allow content editors to wireframe their websites based on a set of standard building blocks. All without needing a developer to lift a finger. This paves the way for low budget websites that have a low dependency on developers.

As Payload is a headless CMS, this is probably not in the scope of the project at this moment. But I think it would accelerate adoption.

**Comment by seaj1one** [comment]
I'd second this. 

Not use if this is possible, but  [https://builderius.io/](url) use GraghQL instead of Rest API. Could you manage the content in Payload CMS and pull it into the above WordPress theme?

Builderius allows you to import any Javascript library too. 


A couple of benefits are to show WordPress developers the possibilities of using Payload and also generate awareness.
